File: Leetcode/LeetCode 75 Study Plan/Day 1 Prefix Sum/1480. Running Sum of 1d Array/solution2.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        running_total = 0
        running_sum = []
        for num in nums:
            running_total += num
            running_sum.append(running_total)
        return running_sum

File: Leetcode/LeetCode 75 Study Plan/Day 1 Prefix Sum/1480. Running Sum of 1d Array/solution2.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        
        int running_total = 0;
        vector<int> running_sum;

        for (int num : nums) {
            running_total += num;
            running_sum.push_back(running_total);
        }

        return running_sum;
        }
};